ios/Podfile.lock

```diff
@@ -12,4 +12,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: cfc2e954ae5b8be9969987641ad09d31bfda9b8b
 
-COCOAPODS: 1.4.0
+COCOAPODS: 1.3.1
```

Comment:
why the downgrade? I think I saw a commit from @albertoestarrona who upgraded it a few days ago. Just want to make sure we don't break functionality somewhere

Reply:
Never mind